app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -145,22 +145,7 @@ class BloodSugarEntrySheet : BottomSheetActivity(), BloodSugarEntryUi, RemoveBlo
 
```

Comment:
Just an FYI, we usually name the commit `Remove unused resources` if we are removing something from the `res` folder like layouts, strings, color, etc., We usually name this as `Remove unused code in -file-name-`